stdeb/util.py

```diff
@@ -130,6 +130,8 @@ def check_call(*popenargs, **kwargs):
      'debian/control Package: (Default: python3-<debianized-setup-name>)'),
     ('suite=',None,
      'suite (e.g. stable, lucid) in changelog (Default: unstable)'),
+    ('suite3=',None,
+     'suite3 Suites used when building with python3 only. Will fallback to Suite if omitted.'),
```

Comment:
Nitpick: how about matching the doc of `Suite` and other Python 3 options more closely:

```
    ('suite3=',None,
     'suite3 (e.g. stable, lucid) in changelog when building with python3 '
     'only, (Default: the value of the suite option)'),
```

Reply:
Defaults with `Default:` are parsed to produce the default value: https://github.com/astraw/stdeb/blob/26d44b10243b75d3bc14bd5062e1e8ce9e79dfeb/stdeb/util.py#L1175-L1183

Since we want suite3's default value to be the value of suite and not its own default we would need to add an additional special case. I can do that if it's preferred.

Reply:
Wow, I didn't know these are extracted automatically.

> we want suite3's default value to be the value of suite

I agree, that would be best to maintain backwards behavior. Otherwise package not declaring `Suite3` (which is pretty much all existing use cases) would after this change use `unstable` for the Python 3 package, right?

Reply:
> Otherwise package not declaring `Suite3` (which is pretty much all existing use cases) would after this change use `unstable` for the Python 3 package, right?

Yeah. Or whatever we write literally after `Default:` which is why I deviated from it in order to get the fallback default of '' and check for that down the road.

https://github.com/astraw/stdeb/blob/26d44b10243b75d3bc14bd5062e1e8ce9e79dfeb/stdeb/util.py#L1206-L1208

It doesn't seem like we want to try and get a config value while setting the defaults, so if we want to use a parsed special case I think we could special case it and then set it to '' explicitly, which would keep other code unchanged from this PR.

Reply:
Anything which makes it work that way is fine with me.

Reply:
e95e06d adds a special case to the parser so the conventional format can be used.